app/presenters/publishing_api/worldwide_corporate_information_page_presenter.rb

```diff
@@ -24,13 +24,17 @@ def content
       content.merge!(
         document_type:,
         schema_name: "worldwide_corporate_information_page",
+        links: {
+          parent: [item.worldwide_organisation.content_id],
+          worldwide_organisation: [item.worldwide_organisation.content_id],
+        },
```

Comment:
I don't have much context on this, but I'm guessing this is the expected format for/location of edition links?

Reply:
Yeah, the edition links exist within the content item, which is sent to Publishing API as part of the `put_content` request.  An example is in the `WorldwideOrganisationPresenter`: https://github.com/alphagov/whitehall/blob/9b9656da2c5e1025d0d727e6cc03fbd5e5c79998/app/presenters/publishing_api/worldwide_organisation_presenter.rb#L30

Whereas non-edition links exist in a separate method, which is sent to Publishing API as a `patch_links` request.